libs/ml/include/ml/utilities/scaler.hpp

```diff
@@ -17,6 +17,8 @@
 //
 //------------------------------------------------------------------------------
 
+#include "math/tensor/tensor.hpp"
```

Comment:
can you explain why this is needed now but it wasn't before?

Reply:
without this header I get error: scaler.hpp:29:29: error: implicit instantiation of undefined template 'fetch::math::Tensor<signed char, fetch::memory::SharedArray<signed char, 1> >'